scrcpy/src/main/java/io/github/miuzarte/scrcpyforandroid/scrcpy/Scrcpy.kt

```
@@ -0,0 +1,1432 @@
+                        } catch (_: InterruptedException) {
+                            if (activeSession !== session || vStream.closed) {
+                                break
+                            }
+                            Thread.interrupted()
+                        } catch (e: Exception) {
+                            Log.w(TAG, "video reader failed", e)
+                            break
+                        }
+                    }
+                } finally {
+                }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFinallyBlock)